client/auth/user-error-display.tsx

```diff
@@ -16,60 +17,96 @@ export interface UserErrorDisplayProps {
+  const { t } = useTranslation()
+
   switch (error.code) {
     case UserErrorCode.InvalidCredentials:
-      return <span>Incorrect username or password</span>
+      return (
+        <span>
+          <Trans t={t} i18nKey='auth.userErrorDisplay.invalidCredentials'>
+            Incorrect username or password
+          </Trans>
+        </span>
+      )
     case UserErrorCode.AccountBanned:
       return (
         <div>
-          This account has been banned.
-          <BanReason>
-            <Subtitle2>Reason:</Subtitle2>
-            <span>{(error.body as any).reason}</span>
-          </BanReason>
-          <span>
-            The ban will expire at {longTimestamp.format((error.body as any).expiration)}.
-          </span>
+          <Trans t={t} i18nKey='auth.userErrorDisplay.accountBanned'>
+            This account has been banned.
+            <BanReason>
+              <Subtitle2>Reason':</Subtitle2>
+              <span>{(error.body as any).reason}</span>
+            </BanReason>
+            <span>
+              The ban will expire at {longTimestamp.format((error.body as any).expiration)}.
+            </span>
+          </Trans>
         </div>
       )
     case UserErrorCode.SessionExpired:
-      return <span>Session expired</span>
+      return (
+        <span>
+          <Trans t={t} i18nKey='auth.userErrorDisplay.sessionExpired'>
+            Session expired
+          </Trans>
+        </span>
+      )
     case UserErrorCode.SuspiciousActivity:
       return (
         <span>
-          Due to suspicious activity detected on this network, creating accounts on the web is
-          currently disabled. Please download the standalone client to create a new account.
+          <Trans t={t} i18nKey='auth.userErrorDisplay.suspiciousActivity'>
+            Due to suspicious activity detected on this network, creating accounts on the web is
+            currently disabled. Please download the standalone client to create a new account.
+          </Trans>
         </span>
       )
     case UserErrorCode.MachineBanned:
       // TODO(tec27): Show expiration date?
-      return <span>This machine is banned from creating new accounts.</span>
+      return (
+        <span>
+          <Trans t={t} i18nKey='auth.userErrorDisplay.machineBanned'>
+            This machine is banned from creating new accounts.
+          </Trans>
+        </span>
+      )
     case UserErrorCode.UsernameTaken:
-      return <span>A user with that name already exists</span>
+      return (
+        <span>
+          <Trans t={t} i18nKey='auth.userErrorDisplay.usernameTaken'>
+            A user with that name already exists
+          </Trans>
+        </span>
+      )
 
     default:
       return (
         <span>
-          An error occurred: {error.status} {error.statusText}
+          <Trans t={t} i18nKey='auth.userErrorDisplay.defaultError'>
+            An error occurred: {error.status} {error.statusText}
```

Comment:
I would guess this is probably the issue with the extractor and that you need to assign status/statusText to variables first

Reply:
The `Trans` examples also use double braces (`{{name}}`) and you've used single throughout this stuff

Reply:
You're also supposed to pass these values in on the `Trans` component, so yeah, this just isn't being used correctly here and is almost certainly the issue, just look at the examples: https://react.i18next.com/latest/trans-component

Reply:
https://react.i18next.com/latest/trans-component#using-with-react-components